.circleci/config.yml

```diff
@@ -292,6 +293,8 @@ jobs:
   docs:
     docker: *docker
     steps:
+      - queue/until_front_of_line:
+          only-on-branch: main
```

Comment:
By default this waits 10m before giving up and I assume failing the job, should we consider extending this or having it wait indefinitely? I think we can monitor here and adjust as needed